HousingSearchApi.Tests/V2/E2ETests/PersonSearchTests.cs

```diff
@@ -91,35 +92,23 @@ public async Task SearchTenure_PaymentRef()
         // Assert
         response.StatusCode.Should().Be(HttpStatusCode.OK);
         var root = GetResponseRootElement(response);
-        root.GetProperty("total").GetInt32().Should().BeGreaterThan(0);
-        var firstResult = root.GetProperty("results").GetProperty("persons")[0];
-        firstResult.GetProperty("tenures")[0].GetProperty("paymentReference").GetString().Should().Be(searchText);
+        try
+        {
+            root.GetProperty("total").GetInt32().Should().BeGreaterThan(0);
+            var firstResult = root.GetProperty("results").GetProperty("persons")[0];
+            firstResult.GetProperty("tenures")[0].GetProperty("paymentReference").GetString().Trim().Should().Be(searchText.Trim());
+        }
+        catch (AssertionException e)
+        {
+            var errMsg = $"Failed to assert that the payment reference is {searchText}";
+            throw new AssertionException(errMsg + "\n" + e.Message);
+        }
     }
 
     # endregion
 
     # region Person
 
-    [Fact]
-    public async Task SearchPerson_Id()
-    {
-        // Arrange
-        var person = RandomItem();
-        var expectedReturnedId = person.GetProperty("id").GetString();
-        var searchText = person.GetProperty("id").GetString();
-        var request = CreateSearchRequest(searchText);
-
-        // Act
-        var response = await _httpClient.SendAsync(request);
-
-        // Assert
-        response.StatusCode.Should().Be(HttpStatusCode.OK);
-        var root = GetResponseRootElement(response);
-        root.GetProperty("total").GetInt32().Should().BeGreaterThan(0);
-        var firstResult = root.GetProperty("results").GetProperty("persons")[0];
-        firstResult.GetProperty("id").GetString().Should().Be(expectedReturnedId);
-    }
-
```

Comment:
We don't need to support this officially - it's a nice to have feature just for devs